test/e2e/test_transformer_inference_service.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 kubeflow.org.
```

Comment:
should we call this test_transformer? Otherwise the test naming might get out of hand.

Reply:
fixed